admin/server/users.go

```diff
@@ -321,6 +332,48 @@ func (s *Server) RevokeUserAuthToken(ctx context.Context, req *adminv1.RevokeUse
 	return &adminv1.RevokeUserAuthTokenResponse{}, nil
 }
 
+// findUserAuthTokenFuzzy attempts to find a user auth token by exact ID, full token string, or unique prefix.
+func (s *Server) findUserAuthTokenFuzzy(ctx context.Context, input string) (*database.UserAuthToken, error) {
+	token, err := s.admin.DB.FindUserAuthToken(ctx, input)
+	if err == nil {
+		return token, nil
+	}
+
+	parsed, err := authtoken.FromString(input)
+	if err == nil {
+		token, err := s.admin.DB.FindUserAuthToken(ctx, parsed.ID.String())
+		if err == nil {
+			return token, nil
+		}
+	}
+
+	if len(input) < 6 {
+		return nil, status.Error(codes.InvalidArgument, "token prefix too short (min 6 chars)")
+	}
+
+	tokens, err := s.admin.DB.FindUserAuthTokens(ctx, "", "", 100)
```

Comment:
Does this work? It's passing `""` as the user ID. Would have to be `claims.OwnerID()`.

The fuzzy logic here is a bit hacky, I'm okay with just supporting full IDs or full tokens if you prefer.

If you want to keep it, then let's add tests for the various match cases.